docs/self-managed/zeebe-deployment/operations/backups.md

```diff
@@ -56,33 +63,34 @@ zeebe-backup-2021-01-31
 └── zeebe-broker-2-data.tar.gz
 ```
 
-
 ### Resuming
 
 After taking the backup, brokers can be started again and will automatically resume with processing.
 
 ## Restore from backup
 
-### Prepare Replacement cluster
+### Prepare replacement cluster
 
-:::caution
+:::note
```

Comment:
Great question. I'll follow up with Amara to confirm, but currently we're certain that only the note admonition follows our current color guidelines and standards.

Reply:
The nice thing here regardless is we can customize a title for the note, which could read "Caution" or "Danger". For example:

```
:::note Your Title

Some **content** with _markdown_ `syntax`.

:::
```

Reply:
@christinaausley Hey, has there been progress on "I'll follow up with Amara to confirm"? Otherwise I'll merge this with a note and custom title as you suggested.